### PR TITLE
ldc: Define CppRuntime on Musl

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

File: ldc/0005-Define-CppRuntime_Clang-on-Musl.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+diff --git a/driver/main.cpp b/driver/main.cpp
+index f119e2d3..cfab07a5 100644
+--- a/driver/main.cpp
++++ b/driver/main.cpp
+@@ -778,6 +778,7 @@ void registerPredefinedTargetVersions() {
+       VersionCondition::addPredefinedGlobalIdent("CRuntime_Bionic");
+     } else if (triple.isMusl()) {
+       VersionCondition::addPredefinedGlobalIdent("CRuntime_Musl");
++      VersionCondition::addPredefinedGlobalIdent("CppRuntime_Clang");
```
Comment (omerfirmak) [inline ldc/0005-Define-CppRuntime_Clang-on-Musl.patch:20]: Alpine does not have Clang runtime tho?

Comment (Geod24) [reply ldc/0005-Define-CppRuntime_Clang-on-Musl.patch]: This is embarassing

Comment (omerfirmak) [reply ldc/0005-Define-CppRuntime_Clang-on-Musl.patch]: for Alpine, yes

**Comment by Geod24** [comment]
Fixed: Using `CppRuntime_Gcc` now.

**Comment by Geod24** [comment]
Upstreamed: https://github.com/ldc-developers/ldc/pull/3769

**Pull request merged**